components/src/web-components/input/gs-date-range-selector.tsx

```diff
@@ -91,7 +91,7 @@ export class DateRangeSelectorComponent extends PreactLitAdapter {
     /**
      * The width of the component.
      *
-     * Visit https://genspectrum.github.io/dashboards/?path=/docs/components-size-of-components--docs for more information.
+     * Visit https://genspectrum.github.io/dashboards-components/?path=/docs/components-size-of-components--docs for more information.
```

Comment:
```suggestion
     * Visit https://genspectrum.github.io/dashboard-components/?path=/docs/components-size-of-components--docs for more information.
```

(also for the links in the other files)

Reply:
Ah, the second renaming... Good catch.